asset-registry/src/tests.rs

```diff
@@ -0,0 +1,377 @@
+#![cfg(test)]
```

Comment:
Unit tests should cover all pallet errors, the following is not covered yet:
- AssetNotFound
- BadVersion
- InvalidAssetId

Reply:
It's not so easy to test the BadVersion error though - that's the error you get when converting locations fails. I am not even sure it is possible.. It's also not being tested in the rest of the project